os/arch/arm/src/amebasmart/Kconfig

```diff
@@ -164,8 +164,13 @@ menu "Realtek RTL8730E BLE Support"
 config AMEBASMART_BLE
 	bool "Enable Amebasmart BLE"
 	default n
+	select FTL_ENABLED
 
 if AMEBASMART_BLE
+config FTL_ENABLED
```

Comment:
How about `amebasmart_ftl_enabled`?

Reply:
FTL_ENABLED is used in TP1x as well, I think it should be consistent with TP1x. If you want it to be amebasmart_ftl_enabled, I can change it too.

Reply:
I have modified it as you have suggested in AIdual, for TP1x I will make the changes in a separate PR.